providers/db.rb

```diff
@@ -39,7 +39,7 @@
 	        :pma_db => new_resource.pma_database,
 	        :auth_type => new_resource.auth_type
 		})
-		mode 00640
+		mode 00644
```

Comment:
It didn't work without it, it was the reason why I could not login into PMA (this file was not readable), I might need to review the PMA owner and group, so it IS accessible for a more limited audience.